### PR TITLE
add schema name as a sequelize storage option.

### PR DESCRIPTION
**Pull request by jonnolen**
will update tests..

### CODE REVIEW

File: lib/storages/sequelize.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ var redefine  = require('redefine');
  * to create if "model" option is not supplied
  * @param {String} [options.storageOptions.tableName=modelName] name of table
  * to create if "model" option is not supplied
+ * @param {String} [options.storageOPtions.schema=schema] name of the schema to create
```
Comment (rpaterson) [inline lib/storages/sequelize.js]: case typo

File: lib/storages/sequelize.js [hunk 2]
```diff
@@ -27,6 +27,8 @@ var redefine  = require('redefine');
...
+ * the table under, defaults to undefeined.
```
Comment (rpaterson) [inline lib/storages/sequelize.js]: "undefeined" -> "undefined"

**Comment by rpaterson** [comment]
LGTM

**Comment by jonnolen** [comment]
fixed typos with last commit.

**Pull request merged**